test/postman/dbp-test-collection.json

```diff
@@ -810,360 +821,98 @@
-					},
-					"response": []
-				},
-				{
-					"name": "Create task instance as testuser",
-					"event": [
-						{
-							"listen": "test",
-							"script": {
-								"id": "e9bd1f06-b857-41b7-ac9d-7b87082b0301",
-								"exec": [
-									"pm.test(\"Status code is 200\", function () {",
-									"    pm.response.to.have.status(200);",
+									"pm.test(\"Token is bearer token\", function () {",
+									"    pm.expect(jsonData.token_type).to.eql(\"bearer\");",
 									"});",
 									"",
-									"var jsonData = pm.response.json();",
-									"pm.test(\"Response contains task id\", function () {",
-									"    pm.expect(\"id\" in jsonData).to.be.true;",
-									"    pm.expect(jsonData.id).to.not.be.null;",
+									"pm.test(\"Response contains access token\", function () {",
+									"    pm.expect(\"access_token\" in jsonData).to.be.true;",
 									"});",
 									"",
-									"pm.globals.set(\"apsTaskId\", jsonData.id);"
-								],
-								"type": "text/javascript"
+									"var tokens = JSON.parse(responseBody); ",
+									"pm.environment.set(\"kcAdminAccessToken\", tokens.access_token);",
+									""
+								]
 							}
 						}
 					],
 					"request": {
 						"method": "POST",
 						"header": [
-							{
-								"disabled": false,
-								"key": "Authorization",
-								"value": "Bearer {{kcAccessToken}}"
-							},
-							{
-								"disabled": false,
-								"key": "Accept",
-								"value": "application/json"
-							},
 							{
 								"key": "Content-Type",
-								"name": "Content-Type",
-								"type": "text",
-								"value": "application/json"
+								"value": "application/x-www-form-urlencoded"
 							}
 						],
 						"body": {
-							"mode": "raw",
-							"raw": "{ \"name\": \"Test Task\" }"
+							"mode": "urlencoded",
+							"urlencoded": [
+								{
+									"key": "client_id",
+									"value": "{{identityServiceClientId}}",
+									"type": "text"
+								},
+								{
+									"key": "grant_type",
+									"value": "password",
+									"type": "text"
+								},
+								{
+									"key": "username",
+									"value": "{{adminUserUsername}}",
+									"type": "text"
+								},
+								{
+									"key": "password",
+									"value": "{{adminUserPassword}}",
+									"type": "text"
+								}
+							]
 						},
 						"url": {
-							"raw": "{{apsUrl}}/api/enterprise/tasks",
+							"raw": "{{protocol}}://{{identityServiceUrl}}/realms/{{identityServiceRealm}}/protocol/openid-connect/token",
+							"protocol": "{{protocol}}",
 							"host": [
-								"{{apsUrl}}"
+								"{{identityServiceUrl}}"
 							],
 							"path": [
-								"api",
-								"enterprise",
-								"tasks"
+								"realms",
+								"{{identityServiceRealm}}",
+								"protocol",
+								"openid-connect",
+								"token"
 							]
 						}
 					},
 					"response": []
 				},
 				{
-					"name": "Retrieve testuser's task instance",
+					"name": "getProcessInstances",
```

Comment:
Can this name be a little bit more descriptive?

Reply:
Done